src/lib/circuit_breaker/circuit_breaker_params.c

```diff
@@ -86,6 +86,21 @@ PARAM_DEFINE_INT32(CBRK_RATE_CTRL, 0);
  */
 PARAM_DEFINE_INT32(CBRK_IO_SAFETY, 0);
 
+/**
+ * Circuit breaker for pre-armed state
+ *
+ * Setting this parameter to 642950 will lock all outputs to failsafe position when prearmed, until armed.
+ * Use together with CBRK_IO_SAFETY to prevent all actuator movement when the safety switch is disabled.
+ * WARNING: ENABLING THIS CIRCUIT BREAKER IS AT OWN RISK
```

Comment:
Do I understand that the behavior is safer if this circuit breaker is enabled? In that case it seems that the logic is inverted from how a circuit breaker is supposed to work.

Reply:
Yes you are right.
Do you suggest to lock actuators by default, and unlock if the circuit breaker is set?  (this would modify the default px4 behaviour)

Reply:
Agree with Julian, I would prefer the opposite. You can also activate the circuit breaker by default to avoid changing the default behaviour of PX4.

Reply:
I find the circuit breakers quite confusing when they are enabled by default though. At this point, I think, it makes more sense to make a general parameter with a certain feature.

Reply:
Ok, I rebased